bokeh/tests/test_io.py

```diff
@@ -105,11 +105,6 @@ def test_default_filename(self):
         filename, resources, title = io._get_save_args(io._state, None, "resources", "title")
         self.assertEqual(filename, "filename")
 
-    def test_missing_filename(self):
```

Comment:
This test didn't actually test for a missing filename (which now is handled without raising an exception and tested elsewhere). It just happened to raise a RuntimeError related to trying to save an object of `type(str)`.